python/hail/methods/impex.py

```diff
@@ -953,6 +953,140 @@ def import_table(paths, key=[], min_partitions=None, impute=False, no_header=Fal
                                    no_header, impute, quote, rg._jrep)
     return Table(jt)
 
+@handle_py4j
+@typecheck(paths=oneof(strlike, listof(strlike)),
+           row_fields=dictof(strlike, Type),
+           key=oneof(strlike, listof(strlike)),
+           entry_type=nullable(Type),
+           missing=strlike,
+           min_partitions=nullable(int),
+           no_header=bool,
+           force_bgz=bool)
+def import_matrix_table(paths, row_fields={}, key=[], entry_type=None, missing="NA", min_partitions=None, no_header=False, force_bgz=False):
```

Comment:
I didn't see any documentation of the field name given to the elements. Can you add documentation for that (or make it a parameter)?

Reply:
what do you mean by elements?

Reply:
the `x`, right?

Reply:
I'll document.